EIPS/eip-3475.md

```diff
@@ -388,38 +384,37 @@ The `_amount` is the amount of the bond for which the spender is approved.
 
 ## Backwards Compatibility
 
-Any contract that inherits the interface of this EIP is compatible. Also, it is valid for both the issuer and receiver of the bonds, and the client EOA needs this standard interface.
+Any contract that inherits the interface of this EIP  is compatible. This  compatibility exists for issuer and receiver of the bonds. Also any client EOA wallet can be compatible with the standard if they are able to sign `issue()` and `redeem()` commands.
 
 However, any existing [EIP-20](./eip-20.md) token contract can issue its bonds by delegating the minting role to a bank contract with the interface of this standard built-in. Check out our reference implementation for the correct interface definition.
 
-To ensure the reading of transactions, "Issue", "Redeem", "Burn" and "Transfer", Events cited above MUST be emitted when such transaction is passed.
+To ensure the reading of transactions, "Issue", "Redeem", "Burn" and "Transfer", events cited above MUST be emitted when such transaction is passed.
```

Comment:
```suggestion
```
This should be part of the specification. If this isn't in there, please move it.